namer/zk-leader/src/main/java/com/twitter/common/zookeeper/Group.java

```diff
@@ -0,0 +1,728 @@
+// =================================================================================================
```

Comment:
Can you just use https://github.com/twitter/finagle/blob/develop/finagle-serversets/src/main/java/com/twitter/finagle/common/zookeeper/Group.java instead?